pkg/controllers/networkpolicy/networkpolicy.go

```diff
@@ -1149,15 +1149,9 @@ func (a *appliedToSecurityGroup) removeStaleMembers(stales []*types.NamespacedNa
 			if strings.Contains(stale.Name, name) {
 				// remove member np tracker.
 				r.Log.V(1).Info("Remove stale members from SecurityGroup", "Stale", stale, "Name", a.id.Name)
-				if tracker := r.getCloudResourceNPTracker(srcMap[name], false); tracker != nil {
+				if tracker := r.getCloudResourceNpTracker(srcMap[name], false); tracker != nil {
```

Comment:
how is stale member removed now? stimulate it with deleting VM from cloud and see if VMP is removed or not

Reply:
The tracker for the stale member is removed here. Nothing changed w.r.t tracker. It will remove the stale member.
We have the integration test and unit-test for it